python/lib/dcoscli/tests/integrations/test_package.py

```diff
@@ -698,7 +698,7 @@ def test_search():
 
     assert returncode == 1
     assert b'' == stdout
-    assert stderr == b'No packages found.\n'
+    assert b'o packages found' in stderr
```

Comment:
```suggestion
    assert b'No packages found' in stderr
```

Reply:
The issue is that we go from a `No packages found` to a `Error: no packages found`

Reply:
Ah ok, due to the missing "n" I just thought its a typo.

Reply:
```suggestion
    assert b'no packages found' in stderr
```
I'd at least add the "n" to avoid confusion, otherwise LGTM.